experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -36,6 +36,8 @@ import { TreeContext } from "./context";
  * Design and document iterator invalidation rules and ordering rules.
  * Providing a custom iterator type with place anchor semantics would be a good approach.
  *
+ * Editing operations are only valid on trees with the `TreeState.InDocument` `TreeState`.
```

Comment:
As editing is done through fields, it might be better to put this comment on TreeField instead. Also linking TreeState would be good since it contains documentation on what this means.

Reply:
I forgot this is the parent type of TreeField (There were some renames...) so I think this is fine here, but could optionally be moved to TreeField.